app/uk/gov/hmrc/apiplatformorganisation/models/responses.scala

```diff
@@ -67,3 +68,34 @@ object CompaniesHouseCompanyProfile {
 
   implicit val format = OFormat[CompaniesHouseCompanyProfile](reads, writes)
 }
+
+object ErrorCode extends Enumeration {
+  type ErrorCode = Value
+
+  val INVALID_REQUEST_PAYLOAD      = Value("INVALID_REQUEST_PAYLOAD")
+  val UNAUTHORIZED                 = Value("UNAUTHORIZED")
+  val UNKNOWN_ERROR                = Value("UNKNOWN_ERROR")
+  val APPLICATION_NOT_FOUND        = Value("APPLICATION_NOT_FOUND")
+  val SCOPE_NOT_FOUND              = Value("SCOPE_NOT_FOUND")
+  val INVALID_CREDENTIALS          = Value("INVALID_CREDENTIALS")
+  val APPLICATION_ALREADY_EXISTS   = Value("APPLICATION_ALREADY_EXISTS")
+  val SUBSCRIPTION_ALREADY_EXISTS  = Value("SUBSCRIPTION_ALREADY_EXISTS")
+  val USER_ALREADY_EXISTS          = Value("USER_ALREADY_EXISTS")
+  val APPLICATION_NEEDS_ADMIN      = Value("APPLICATION_NEEDS_ADMIN")
+  val CLIENT_SECRET_LIMIT_EXCEEDED = Value("CLIENT_SECRET_LIMIT_EXCEEDED")
```

Comment:
Are all these error codes needed especially the application, subscription and client secret related ones?

Reply:
Good spot. Fixed